CHANGELOG.md

```diff
@@ -12,6 +12,17 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 ### Added
 * New detector `FindInstanceLockOnSharedStaticData` for new bug type `SSD_DO_NOT_USE_INSTANCE_LOCK_ON_SHARED_STATIC_DATA`. This detector reports a bug if an instance level lock is used to modify a shared static data. (See [SEI CERT rule LCK06-J](https://wiki.sei.cmu.edu/confluence/display/java/LCK06-J.+Do+not+use+an+instance+lock+to+protect+shared+static+data))
 
+### Added
+* New detector `MultipleInstantiationsOfSingletons` and introduced new bug types:
+  * `SING_SINGLETON_HAS_NONPRIVATE_CONSTRUCTOR` is reported in case of a non-private constructor, 
+  * `SING_SINGLETON_IMPLEMENTS_CLONEABLE` is reported in case of a class directly implementing the `Cloneable` interface, 
+  * `SING_SINGLETON_INDIRECTLY_IMPLEMENTS_CLONEABLE` is reported when a class indirectly implements the `Cloneable` interface, 
+  * `SING_SINGLETON_IMPLEMENTS_CLONE_METHOD` is reported when a class does not implement the `Cloneable` interface, but has a `clone()` method, 
+  * `SING_SINGLETON_IMPLEMENTS_SERIALIZEABLE` is reported when a class directly or indirectly implements the `Serializable` interface and 
+  * `SING_SINGLETON_GETTER_NOT_SYNCHRONIZED` is reported when the instace-getter method of the singleton class is not synchronized.
```

Comment:
Misspelled "instance"

Also, it's not essential for a singleton-getter to be entirely synchronized; it can do a null-check first before synchronizing, for efficiency.